addons/mail/static/src/models/message.js

```diff
@@ -801,6 +811,10 @@ registerModel({
             compute: '_computeIsBodyEqualSubtypeDescription',
             default: false,
         }),
+        isDiscussion: attr({
```

Comment:
We should at least comment both field to avoid confusion.
But we could also rename this one to `isDiscussionOrNotification` and avoid the burden of the comments.

This field look mainly used to change the message component style, so `isDiscussionLayout` could be a other option ?

Reply:
Change done.